js/prefs/switchencryptionoptions.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Managing Keys for identities in the preferences UI.
+ *
+ * @author     Rafael te Boekhorst <boekhorst@b1-systems.de>
+ * @copyright  2014-2015 Horde LLC
+ * @license    GPL-2 (http://www.horde.org/licenses/gpl)
+ */
+
+var switchEncryption = {
+
+    // checks wheather the smime checkbox is clicked and shows/hides
+    checkSMIMEOption: function () {
+        // get checkbox info
+        if ($('smimeselect').checked === true) {
```

Comment:
Use document.getElementById(idstring) for this type

Reply:
Ok I replaced all prototypejs to normal javascript. Thanks for the email with information! 

I think some functionality has now decreased. If I click the check box "Check if you would like to use SMIME-keypairs for identities", then the SMIME option appears but the infos on identity-keys in the addressbook remain, sometimes (!), after some more clicking they get removed..